Estore.UITests/StepDefinitions/Assertions/CommonStepsAssertions.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Estore.Models.Enum;
+using NUnit.Framework;
+using TechTalk.SpecFlow;
+using UITests.Context;
+
+namespace Estore.UITests.StepDefinitions.Assertions
+{
+    [Binding]
+    public class CommonStepsAssertions
+    {
+        private readonly DataContext _context;
+
+        public CommonStepsAssertions(DataContext context)
+        {
+            _context = context;
+        }
+
+        [Then(@"Info message '([^']*)' is presented")]
+        public void ThenInfoMessageIsPresented(string infoMessage)
+        {
+            Assert.AreEqual(infoMessage, _context.CurrentPage.GetInfoMessage(), "Info messages are not equal");
+        }
+
+        [Then(@"Login button is displayed")]
+        public void ThenLoginButtonIsDisplayed()
+        {
+            Assert.IsTrue(_context.CurrentPage.WaitLoginLinkLoading(),
+                "Login button is not displayed");
+        }
+
+        [Then(@"Navigation bar has next items called (.*)")]
+        public void ThenNavigationBarHasNextItems(string names)
```

Comment:
It is better to pass the array instead of splitting string. Ex: 
<img width="264" alt="image" src="https://github.com/TiAutoBootcamp/User-UI-Test/assets/48569567/99ed25ec-f29a-4dc0-a0cf-996dd12035dc">

Reply:
fixed